lib/questions.rb

```diff
@@ -187,19 +231,27 @@ def word_count_a_file(file_path)
 # called call_method_from_string('foobar')
 # the method foobar should be invoked
 def call_method_from_string(str_method)
+  send(str_method)
 end
 
 # return true if the date is a uk bank holiday for 2014
 # the list of bank holidays is here:
 # https://www.gov.uk/bank-holidays
 def is_a_2014_bank_holiday?(date)
+  bank_holidays = ['2014, 12, 26', '2014, 12, 25', '2014, 08, 25', '2014, 05, 26', '2014, 05, 05', '2014, 04, 21', '2014, 04, 18', '2014, 01, 01']
+  arr = bank_holidays.map {|i| Date.strptime(i, '%Y, %m, %d')}
+  arr.include?(date)
```

Comment:
the passed in `date` is actually an instance of `Time` and so does not match any of the entries in the array; which are all `Date`.  `arr.include?(date.to_date)` will fix it.  Poorly written tests 😉